File: data-jpa/src/test/java/top/bettercode/summer/data/jpa/repository/EmployeeRepository.java
package top.bettercode.summer.data.jpa.repository;

import top.bettercode.summer.data.jpa.JpaExtRepository;
import top.bettercode.summer.data.jpa.domain.Employee;
import top.bettercode.summer.data.jpa.domain.EmployeeKey;

/**
 * @author Peter Wu
 */
public interface EmployeeRepository extends JpaExtRepository<Employee, EmployeeKey> {

}

File: data-jpa/src/test/kotlin/top/bettercode/summer/data/jpa/repository/EmployeeRepository.kt
package top.bettercode.summer.data.jpa.repository

import top.bettercode.summer.data.jpa.JpaExtRepository
import top.bettercode.summer.data.jpa.domain.Employee
import top.bettercode.summer.data.jpa.domain.EmployeeKey

/**
 * @author Peter Wu
 */
interface EmployeeRepository : JpaExtRepository<Employee?, EmployeeKey?>
